sections/Testimonials/Testimonials.{MrtDev}.tsx

```diff
@@ -0,0 +1,62 @@
+export type Testimonial = {
+  logoVector?: string;
+  testimonial: string;
+  person: string;
+  nameSurname: string;
+  positionSurname: string;
+};
+
+interface TestimonialsProps {
```

Comment:
Para aparecer no admin da deco
```suggestion
export interface TestimonialsProps {
```